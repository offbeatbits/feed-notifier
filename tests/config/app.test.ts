describe("App config", () => {
  it("should export app config", () => {
    expect(require("../../src/config/app")).toMatchSnapshot(); // eslint-disable-line
  });
});
