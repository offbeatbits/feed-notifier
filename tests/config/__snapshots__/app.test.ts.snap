// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App config should export app config 1`] = `
Object {
  "cacheDirectory": "test_cache_directory",
  "cacheKey": "article",
  "enableLogging": false,
  "enableVerboseLogging": false,
}
`;
